packages/grafana-ui/src/themes/GlobalStyles/elements.ts

```diff
@@ -35,9 +34,6 @@ export function getElementStyles(theme: GrafanaTheme2) {
     },
 
     body: {
-      height: '100%',
-      width: '100%',
-      position: 'unset',
```

Comment:
With this or display: flex + flexGrow how would we create a page that takes up full height (like panel edit), use absolute positioning instead flex?